### PR TITLE
Add recipe for pyalveo

### PR DESCRIPTION
**Pull request by stevecassidy**
My first PR to conda-forge so hopefully all is well.

### CODE REVIEW

File: recipes/pyalveo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/pyalveo/meta.yaml]: Please delete all comments

File: recipes/pyalveo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/pyalveo/meta.yaml]: It looks like `python` and `setuptools` are the only `build` requirements. Typically those are all you need if you're using `--single-version-externally-managed`, unless the `setup.py` imports them directly or indirectly or if there is a `setup_requires` section.

File: recipes/pyalveo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    - python
+    - setuptools
+    - requests
+    - oauthlib
+    - requests-oauthlib
+  run:
```
Comment (synapticarbors) [inline recipes/pyalveo/meta.yaml]: The `run` section should include:
 - python-dateutil
 - requests
 - oauthlib
 - requests-oauthlib

File: recipes/pyalveo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    - python
+
+test:
+  imports:
+    - pyalveo
+  requires:
```
Comment (synapticarbors) [inline recipes/pyalveo/meta.yaml]: If packages are listed as `run` requirements, they don't need to be in the `requires` section of `test`.  You should be able to delete the entire `requires` section.

File: recipes/pyalveo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    - oauthlib
+    - requests-oauthlib
+    - python-dateutil
+
+about:
+  home: http://github.com/Alveo/pyalveo
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD-3-Clause
```
Comment (synapticarbors) [inline recipes/pyalveo/meta.yaml]: I couldn't find any reference to a particular version of `BSD`. I've raised an issue upstream:

https://github.com/Alveo/pyalveo/issues/8

In the mean time, you can add the above link as a comment so we can resolve the license issue later in the feedstock once things are fixed upstream (unless it's fixed upstream before we merge this).

**Comment by bgruening** [comment]
Thanks @stevecassidy!

**Pull request merged**